build.gradle

```diff
@@ -1,47 +1,21 @@
-//////////////////////////////
-// Buildscript Dependencies //
-//////////////////////////////
-buildscript {
-    repositories {
-        jcenter()
-        maven { url 'https://palantir.bintray.com/releases/' }
-    }
-
-    dependencies {
-        classpath 'com.netflix.nebula:gradle-info-plugin:7.1.4'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.5'
-        classpath 'com.netflix.nebula:nebula-publishing-plugin:17.2.1'
-    }
-}
-
 /////////////
 // Plugins //
 /////////////
 plugins {
     id 'com.palantir.git-version' version '0.12.3'
     id 'org.inferred.processors' version '3.3.0'
-}
-
-apply plugin: 'java'
-apply plugin: 'idea'
+    id 'com.palantir.external-publish' version '0.4.0'
+    id 'com.palantir.external-publish-jar' version '0.4.0'
 
-// intentionally not applying nebula.maven-publish, but most of its constituent
-// plugins, because we do _not_ want nebula.maven-compile-only
-apply plugin: 'com.jfrog.bintray'
-apply plugin: 'maven-publish'
-apply plugin: 'nebula.maven-nebula-publish'
-apply plugin: 'nebula.maven-base-publish'
-apply plugin: 'nebula.maven-developer'
-apply plugin: 'nebula.maven-manifest'
-apply plugin: 'nebula.info-scm'
-apply plugin: 'nebula.maven-scm'
-apply plugin: 'nebula.source-jar'
+    id 'java'
+    id 'idea'
+}
 
 /////////////////////////
 // Module Dependencies //
 /////////////////////////
 repositories {
-    jcenter()
```

Comment:
nit: you could still use jcenter until 2022. If you don't have any deps there, then I think its reasonable to remove it.

Reply:
Yeah, only dep total is just junit, so it is super easy to switch repos.